requirements_dev.txt

```diff
@@ -7,11 +7,15 @@ tox==3.14.0
 coverage==5.1
 Sphinx==1.8.5
 twine==1.14.0
-Click==7.0
-setuptools==45.1.0
+Click>=7.0
```

Comment:
same as before. Why are we using >=, rather than == ?

Reply:
We will specify minimum version number required for the package unless the code breaks, in which case we will create an issue and update the code. Until the code is fixed the version number for that particular package can be pinned.